docs/running-genesis-node.md

```diff
@@ -0,0 +1,161 @@
+    This means that the node in updating to the latest network state (it takes some time).
+        
+        You can also check node status by executing the command `dclcli status` to get the current status.
+        The value of `latest_block_height` reflects the current node height.
+       
+6. Check that genesis account is created:
+
+    * In order to ensure that account is created and has assigned role you can use the command: 
+    `dclcli query auth account --address=<address>`.
+    Expected output format: 
+        ```json
+        {
+          "result": {
+            "address": string, // bench32 encoded address
+            "public_key": "string, // bench32 encoded public key
+            "roles": [
+              "NodeAdmin"
+            ],
+            "coins": [],
+            "account_number": string,
+            "sequence": string
+          },
+          "height": string
+        }
+        ```
+
+6. Check the node is running and participates in consensus:
+    * Get the list of all nodes: `dclcli query validator all-nodes`. 
+    The node must present in the list and has the following params: `power:10` and `jailed:false`.
+
+    * Get the node status: `dclcli status --node <node ip>`. 
+    The value of `node ip` matches to `[rpc] laddr` field in `$HOME/.dcld/config/config.toml`
+    (TCP or UNIX socket address for the RPC server to listen on).  
+    Make sure that `result.sync_info.latest_block_height` is increasing over the time (once in about 5 sec).
+       Expected output format: 
+        ```json
+        {
+          "node_info": {
+            "protocol_version": {
+              "p2p": "7",
+              "block": "10",
+              "app": "0"
+            },
+            "id": string, // matches to prefix <ID> of the file: $HOME/.dcld/config/gentx/gentx-<ID>.json
+            "listen_addr": "tcp://0.0.0.0:26656", // Address to listen for incoming connections. Matches to $HOME/.dcld/config/config.toml [p2p] `laddr` filed.
+            "network": "dclchain",
+            "version": "0.32.8",
+            "channels": string,
+            "moniker": string,
+            "other": {
+              "tx_index": "on",
+              "rpc_address": "tcp://127.0.0.1:26657" // TCP or UNIX socket address for the RPC server to listen on. Matches to $HOME/.dcld/config/config.toml [rpc] `laddr` filed. 
+            }
+          },
+          "sync_info": {
+            "latest_block_hash": string,
+            "latest_app_hash": "string,
+            "latest_block_height": string,
+            "latest_block_time": string,
+            "catching_up": bool
+          },
+          "validator_info": {
+            "address": string,
+            "pub_key": {
+              "type": string,
+              "value": string
+            },
+            "voting_power": string
+          }
+        }
+        ```
+    
+    * Get the list of nodes participating in the consensus for the last block: `dclcli tendermint-validator-set`.
```

Comment:
Should we do this step only after all the genesis nodes are started up (finish the steps above)?